new-lamassu-admin/src/pages/Analytics/components/wrappers/OverTimeWrapper.js

```diff
@@ -0,0 +1,79 @@
+import { Box } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import React from 'react'
+
+import { Select } from 'src/components/inputs'
+import { H2 } from 'src/components/typography'
+import { primaryColor } from 'src/styling/variables'
+
+import styles from '../../Analytics.styles'
+import Graph from '../../graphs/Graph'
+import LegendEntry from '../LegendEntry'
+
+const useStyles = makeStyles(styles)
+
+const OverTimeDotGraphHeader = ({
+  title,
+  representing,
+  period,
+  data,
+  machines,
+  selectedMachine,
+  handleMachineChange,
+  timezone,
+  currency
+}) => {
+  const classes = useStyles()
+
+  const legend = {
+    cashIn: <div className={classes.cashInIcon}></div>,
+    cashOut: <div className={classes.cashOutIcon}></div>,
+    transaction: <div className={classes.txIcon}></div>,
+    average: (
+      <svg height="12" width="18">
+        <g fill="none" stroke={primaryColor} strokeWidth="3">
+          <path strokeDasharray="5, 2" d="M 5 6 l 20 0" />
+        </g>
+      </svg>
+    )
+  }
+
+  return (
+    <>
+      <div className={classes.graphHeaderWrapper}>
+        <div className={classes.graphHeaderLeft}>
+          <H2 noMargin>{title}</H2>
+          <Box className={classes.graphLegend}>
+            <LegendEntry IconElement={legend.cashIn} label={'Cash-in'} />
+            <LegendEntry IconElement={legend.cashOut} label={'Cash-out'} />
+            <LegendEntry
+              IconElement={legend.transaction}
+              label={'One transaction'}
+            />
```

Comment:
According to the spec, it's just to mean that a dot symbolizes one transaction (either cash-in or out). But it might get confusing with the coloring, I agree